### PR TITLE
Latejoin crabbans shouldn't get stuck at the menu anymore

### PR DESCRIPTION
**Pull request by ghost**
I just turn them into crabs 5 seconds after `hippie_after_spawn`

### CODE REVIEW

File: hippiestation/code/modules/jobs/job_types/job.dm [hunk 1]
```diff
@@ -2,6 +2,8 @@
 	if(!H || !M)
 		return FALSE
 	if(is_banned_from(M.ckey, CRABBAN))
-		SSticker.OnRoundstart(CALLBACK(H, /mob/living/carbon/human.proc/change_mob_type, /mob/living/simple_animal/crab, null, H.real_name, TRUE)) // where's my antag token
-	else if(is_banned_from(M.ckey, CATBAN))
+		if(!SSticker.HasRoundStarted())
+			SSticker.OnRoundstart(CALLBACK(H, /mob/living/carbon/human.proc/change_mob_type, /mob/living/simple_animal/crab, null, H.real_name, TRUE)) // where's my antag token
+		ekse
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/jobs/job_types/job.dm]: You didn't test this, did you?

**Review by Manly-Dwarf — CHANGES_REQUESTED** [review]

**Review by Manly-Dwarf — APPROVED** [review]
Looks fine now, and tested.

**Comment by yoyobatty** [comment]
I feel like this entire piece of code is hacky, but whatever.

**Comment by ghost** [comment]
travis is hung forever because byond down, btw

**Pull request merged**